packages/components/src/standalone-comparison-grid/index.stories.jsx

```diff
@@ -0,0 +1,46 @@
+import StandaloneComparisonGrid from '.';
+import './stories.scss';
+
+export default { title: 'packages/components/StandaloneComparisonGrid' };
+
+const columns = [
+	{
+		title: 'Import',
+		rows: [
+			'Import your pages and posts',
+			'Import your uploaded images and files',
+			'Import your users**',
+			'Import your menus** and FSE templates**',
+		],
+		intro_copy:
```

Comment:
Tip: Javascript/Typescript usually relies more on camelCase than the regular PHP snake_case